keep-ui/.env.local.example

```diff
@@ -14,4 +14,6 @@ PUSHER_HOST=localhost
 PUSHER_PORT=6001
 PUSHER_APP_KEY=keepappkey
 # Logging
-LOG_FORMAT=dev_terminal
\ No newline at end of file
+LOG_FORMAT=dev_terminal
+
+DISABLE_POSTHOG=true
```

Comment:
```suggestion
```

Reply:
Why do you suggest this? I added it to disable Posthog for dev envs. The same with the previous change.